src/effects/audiounits/AudioUnitEffect.cpp

```diff
@@ -1831,8 +1861,12 @@ TranslatableString AudioUnitEffect::Export(const wxString & path) const
 }
 
 std::pair<CF_ptr<CFDataRef>, TranslatableString>
-AudioUnitEffect::MakeBlob(const wxCFStringRef &cfname, bool binary) const
+AudioUnitEffect::MakeBlob(const AudioUnitEffectSettings &settings,
+   const wxCFStringRef &cfname, bool binary) const
 {
+   // Update state of the unit from settings
+   StoreSettings(settings);
+
```

Comment:
The commit message says "...And use it in SavePreset" but it is actually used only here (?)